cmd/ocm/login/cmd.go

```diff
@@ -239,6 +247,7 @@ func run(cmd *cobra.Command, argv []string) error {
 
 	// Update the configuration with the values given in the command line:
 	cfg.TokenURL = tokenURL
+	cfg.RhRegion = args.rhRegion
```

Comment:
again, no need to persist the `RhRegion` to the config. We go from args.rhRegion -> region discover -> cfg.URL, once we have the URL we are done with region. If we find a specific need later where we need to keep track of the name of the region for some purpose, we can add it to the persisted config then. For now, I dont see it being needed after line 284 (`cfg.URL = regValue.URL`)

Reply:
Ack. Removed